routes/api.php

```diff
@@ -31,3 +32,28 @@
     Route::put('/{user}', 'UserController@update')->name('update');
     Route::delete('/{user}', 'UserController@destroy')->name('destroy');
 });
+
+// Jacky here
+Route::group([
```

Comment:
You can make nested groups the large one to contain the common between all routes
```
Route::group([
    'namespace' => 'App\\Http\\Controllers\\API\\V1\\',
    'prefix' => '/V1',
    'as' => 'api.v1.'
], function () {
      Route::group([
        'as' => 'task.'
      ], function () {
         .....
      });
    
     Route::group([
        'as' => 'projects.'
      ], function () {
         .....
      });
});
```